components/underlay/index.css

```diff
@@ -13,34 +13,67 @@ governing permissions and limitations under the License.
 @import '../overlay/index.css';
 
 .spectrum-Underlay {
-  /* Bug: this must be 0ms, not 0 */
-  --spectrum-dialog-confirm-background-entry-animation-delay: 0ms;
-  --spectrum-dialog-confirm-background-exit-animation-ease: cubic-bezier(0.5, 0, 1, 1); /* wrong in DNA */
-  --spectrum-dialog-confirm-background-entry-animation-ease: cubic-bezier(0, 0, 0.40, 1); /* wrong in DNA */
+  --spectrum-underlay-background-entry-animation-delay: var(--spectrum-animation-duration-0);   /* Bug: this must be 0ms, not 0 */
+  --spectrum-underlay-background-exit-animation-ease: cubic-bezier(0.5, 0, 1, 1); /* wrong in DNA */
```

Comment:
Is there an issue associated with this so we can track it's update?

Reply:
not that I am aware of or that I could find. 

I however was able to replace each of these with our custom animation tokens